docs/intro/tutorial06.txt

```diff
@@ -44,11 +44,11 @@ reference the path for templates.
 
 .. admonition:: Static file namespacing
 
-    Just like templates, we *might* be able to get away with putting our static
-    files directly in ``polls/static`` (rather than creating another ``polls``
-    subdirectory), but it would actually be a bad idea. Django will choose the
-    first static file it finds whose name matches, and if you had a static file
-    with the same name in a *different* application, Django would be unable to
+    We *might* be able to get away with putting our static files directly in
```

Comment:
Didn't see the reason we needed to remove the reference to templates here, reverting in my edits.